docs/openapi/api-internal-pn-bff-tos-privacy.yaml

```diff
@@ -0,0 +1,128 @@
+      - $ref: 'remote-refs.yaml#/components/parameters/cxTypeAuthFleet'            # NO EXTERNAL
+    get:
+      summary: Accettazione TOS e Privacy
+      description: >-
+        Ritorna lo stato di accettazione dei TOS e della Privacy.
+      tags:
+        - UserConsents
+      operationId: getTosPrivacyV1
+#      security:                                      # ONLY EXTERNAL
+#        - bearerAuth: [ ]                            # ONLY EXTERNAL
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BffTosPrivacyConsent'
+        '400':
+          description: Invalid input
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+
+    put:
+      summary: Accettazione TOS e Privacy
+      description: >-
+        Permette di accettare i TOS e la Privacy.
+      tags:
+        - UserConsents
+      operationId: acceptTosPrivacy
+#      security:                                      # ONLY EXTERNAL
+#        - bearerAuth: [ ]                            # ONLY EXTERNAL
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BffTosPrivacyBody'
+      responses:
+        '200':
+          description: successful operation
+        '400':
+          description: Invalid input
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+
+components:
+  parameters:
+    ### PARAMETRI DI AUTENTICAZIONE
+    cxTypeAuthFleet:                                                               # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/cxTypeAuthFleet'            # NO EXTERNAL
+    uidAuthFleet:                                                                  # NO EXTERNAL
+      $ref: './remote-refs.yaml#/components/parameters/uidAuthFleet'               # NO EXTERNAL
+
+  schemas:
+    BffConsentAction:
+      $ref: './remote-refs.yaml#/components/schemas/ConsentAction'
+    BffConsentVersion:
+      $ref: './remote-refs.yaml#/components/schemas/ConsentVersion'
+    BffConsent:
+      $ref: './remote-refs.yaml#/components/schemas/Consent'
+
+    BffTosPrivacyConsent:
+      type: object
+      required:
+        - tos
+        - privacy
+      properties:
+        tos:
+          $ref: '#/components/schemas/BffConsent'
+        privacy:
+          $ref: '#/components/schemas/BffConsent'
+
+    BffTosPrivacyBody:
+      type: object
+      properties:
+        tos:
+          $ref: '#/components/schemas/BffTosPrivacyActionBody'
+        privacy:
+          $ref: '#/components/schemas/BffTosPrivacyActionBody'
+
+    BffTosPrivacyActionBody:
+      type: object
+      required:
+        - action
+        - version
+      allOf:
```

Comment:
Using the allOf, the BffConsentAction and BffConsentVersion are generated but never used (see how BffTosPrivacyActionBody is generated). For this reason remove them and move the ref value directly into BffTosPrivacyActionBody